Formula/dosbox-staging.rb

```diff
@@ -0,0 +1,34 @@
+class DosboxStaging < Formula
+  desc "Modernized DOSBox soft-fork"
+  homepage "https://dosbox-staging.github.io/"
+  url "https://github.com/dosbox-staging/dosbox-staging/archive/v0.75.0.tar.gz"
+  sha256 "6279690a05b9cc134484b8c7d11e9c1cb53b50bdb9bf32bdf683bd66770b6658"
+  license "GPL-2.0"
+  head "https://github.com/dosbox-staging/dosbox-staging.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libpng"
+  depends_on "opusfile"
+  depends_on "sdl2"
+  depends_on "sdl2_net"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-sdltest
+      --enable-core-inline
+    ]
+
+    system "./autogen.sh"
+    system "./configure", *args
+    system "make", "install"
+    mv bin/"dosbox", bin/"dosbox-staging"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/dosbox-staging -version")
```

Comment:
I still feel https://github.com/Homebrew/homebrew-core/pull/54659#discussion_r424669240 is relevant here.

Is there no way to exercise even very basic functionality?

Reply:
Hmm there's a `-printconf` flag that outputs the path to a configuration file named `dosbox-staging.conf`

Other than that there only seems to be a `-editconf` flag that opens the config file in a text editor

Source: https://github.com/dosbox-staging/dosbox-staging/blob/master/src/gui/sdlmain.cpp

Reply:
Nice!
That `-printconf` thing sounds just right.

Reply:
Done ✅ Took me awhile to realize that `testpath` could be treated like `HOME`